biome.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.4.1/schema.json",
+  "formatter": {
+    "enabled": true,
+    "ignore": [
+      "tests/fixtures/sample-esy/esy.lock",
+      "tests/fixtures/sample-esy/_esy"
+    ],
+    "indentStyle": "space"
+  },
+  "linter": {
+    "enabled": true,
+    "ignore": ["astexplorer"],
```

Comment:
There are too many warnings, so take care of them with another PR.

Reply:
Isn't that code vendored anyway? In other words, we can't fix the warnings without making it hard for us to update this.

Reply:
Ah, indeed.